Dockerfile

```diff
@@ -46,4 +46,6 @@ RUN helm plugin install https://github.com/databus23/helm-diff && \
 
 COPY --from=builder /workspace/helmfile/dist/helmfile_linux_amd64 /usr/local/bin/helmfile
 
-CMD ["/usr/local/bin/helmfile"]
+WORKDIR /wd
```

Comment:
This makes sense, but I'd like to keep this `WORKDIR /`, as I've seen someone uses the image like:

```
docker -v ${PWD}/helmfile.yaml:/helmfile.yaml ...
```

And this change can be a breaking change for those.

Reply:
Indeed this is a breaking change for some. However, helmfile supports referencing multiple helmfiles, and mounting one by one can be tricky.